schema/schema.v0.json

```diff
@@ -10,19 +10,64 @@
         "additionalProperties": false,
         "properties": {
             "addMedia": {
-                "title": "addMedia",
-                "type": "array",
-                "items": {
-                    "type": "string",
-                    "pattern": "^\\.\/assets\/.*\\.(png|jpeg|jpg|gif|mp4)$"
-                },
-                "minItems": 1,
-                "description": "List of media file paths to add to the device's gallery",
-                "examples": [
-                    [
-                        "./assets/image1.png",
-                        "./assets/video1.mp4"
-                    ]
+                "oneOf": [
+                    {
+                        "title": "addMedia",
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "pattern": "^.*\\.(png|jpeg|jpg|gif|mp4)$"
```

Comment:
Are the file extensions required?

Reply:
Just followed the api reference https://maestro.mobile.dev/api-reference/commands/addmedia.

```
Currently, maestro supports the following mime types for this command:

PNG

JPEG

JPG

GIF

MP4
```